WordPressAuthenticator/Unified Auth/View Related/Login/MagicLinkRequestedViewController.swift

```diff
@@ -0,0 +1,158 @@
+import UIKit
+import WordPressUI
+
+final class MagicLinkRequestedViewController: LoginViewController {
+
+    // MARK: Properties
+
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var subtitleLabel: UILabel!
+    @IBOutlet private weak var emailLabel: UILabel!
+    @IBOutlet private weak var cannotFindEmailLabel: UILabel!
+    @IBOutlet weak var buttonContainerView: UIView!
+    @IBOutlet weak var loginWithPasswordButton: UIButton!
+
+    private let email: String
+    private let loginWithPassword: () -> Void
+
+    private lazy var buttonViewController: NUXButtonViewController = .instance()
+
+    init(email: String, loginWithPassword: @escaping () -> Void) {
+        self.email = email
+        self.loginWithPassword = loginWithPassword
+        super.init(nibName: "MagicLinkRequestedViewController", bundle: WordPressAuthenticator.bundle)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override var sourceTag: WordPressSupportSourceTag {
+        .loginMagicLink
```

Comment:
Nit:
This being a new screen should we consider adding `sourceTag`? 
This might be helpful when we work on help center experiment. 😅 Internal ref - pe5sF9-n8-p2